sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_client_models.hpp

```diff
@@ -899,4 +905,107 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
         Azure::Core::Context const& context = Azure::Core::Context());
   };
 
+  enum LifetimeActionType
+  {
+    /**
+     * @brief Rotate the key based on the key policy.
+     *
+     */
+    Rotate,
+
+    /**
+     * @brief Trigger event grid events. For preview, the notification time is not configurable
+     * and it is default to 30 days before expiry.
+     *
+     */
+    Notify
+  };
+
+  /**
+   * @brief A condition to be satisfied for an action to be executed.
+   */
+  struct LifetimeActionsTrigger final
+  {
+    /**
+     * @brief Time after creation to attempt to rotate. It only applies to rotate. It will be in
+     * ISO 8601 duration format. Example: 90 days : "P90D"
+     *
+     */
+    Azure::Nullable<std::string> TimeAfterCreate;
```

Comment:
Is this the name used in other languages as well? I wonder it does not mention `rotate` on the name.

Reply:
same names in .Net. These names are from the documentation (swagger)